### PR TITLE
add host unit is free agreement checkbox

### PR DESCRIPTION
**Pull request by beniamin**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Adds a new agreement checkbox in add accommodation form, in order for host user to consent that the hosting unit is completely free.

### CODE REVIEW

File: app/Http/Controllers/StaticPagesController.php [hunk 1]
```diff
@@ -22,9 +22,6 @@ class StaticPagesController extends Controller
      */
     public function home(Request $request, SettingRepository $settingRepository)
     {
-        $helpRequests = HelpRequest::all()->count();
```
Comment (beniamin) [inline app/Http/Controllers/StaticPagesController.php:25]: am facut putin curatenie, nu pare sa fie afisate pe undeva prin front astea, asa ca le-am scos pentru ca se faceau niste query-uri degeaba.

File: app/Accommodation.php [hunk 2]
```diff
@@ -206,11 +206,6 @@ public function getDisplayedAddress(): string
         return implode(', ', $addressComponents);
     }
 
-    public function scopeIsFree(Builder $query): Builder
```
Comment (beniamin) [inline app/Accommodation.php:209]: nu mai este folosita din cauza de cleanup pe homepage.

File: resources/views/frontend/home.blade.php [hunk 3]
```diff
@@ -26,44 +26,6 @@ class="btn btn-lg btn-secondary px-sm-5">{{ __('Offer Accommodation') }}</a>
         </div>
     </div>
 
-    {{-- <div class="container py-5">
```
Comment (beniamin) [inline resources/views/frontend/home.blade.php:29]: niste curatenie pe homepage.

Comment (andreiio) [reply resources/views/frontend/home.blade.php:29]: leave this in, a fost removed doar cu titlu temporar

File: docker-compose.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ services:
     environment:
       PHP_OPCACHE_ENABLE: 0
     volumes:
-      - ./:/var/html:delegated
+      - ./:/var/www:delegated
```
Comment (gheorghelupu17) [inline docker-compose.yml:14]: Pls roll back here

**Review by gheorghelupu17 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 4 above)
Comment (beniamin) [reply docker-compose.yml:14]: e aceeasi modificare care s-a dus si pe main mai devreme putin: https://github.com/code4romania/war-support-un-acoperis/commit/faf5c96fc00e3090ad3819ecfd6cc790ccfe109a#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R14

File: resources/views/frontend/home.blade.php (hunk 3 above)
Comment (beniamin) [reply resources/views/frontend/home.blade.php]: done

**Review by gheorghelupu17 — APPROVED** [review]

**Pull request merged**